vividus-engine/src/main/java/org/vividus/expression/ArgumentMatcher.java

```diff
@@ -16,12 +16,9 @@
 
 package org.vividus.expression;
 
-import org.apache.commons.lang3.StringUtils;
-
-interface NormalizingArguments
+public interface ArgumentMatcher
```

Comment:
no need to use interface here, most likely there won't be multiple implementations